classes/actions/ShoppingfeedOrderImportActions.php

```diff
@@ -1303,4 +1299,41 @@ protected function initCarrierFinder()
     {
         return new CarrierFinder();
     }
+
+    private function checkQtyProduct($id_product, $id_product_attribute, $qty)
```

Comment:
protected instead of private. This class can be extended by an override or a hook